src/KM_Controls.pas

```diff
@@ -5858,6 +5881,7 @@ procedure TKMDropColumns.ListChange(Sender: TObject);
 
 procedure TKMDropColumns.ListHide(Sender: TObject);
 begin
+  fListTopIndex := fList.TopIndex; //Save list scroll position

```

Comment:
Doesn't this need to be implemented for TKMDropList too? Currently TKMDropList.fListTopIndex is never set but is used in TKMDropList.ListShow